admin/class-admin.php

```diff
@@ -420,68 +420,6 @@ protected function initialize_seo_links() {
 	/********************** DEPRECATED METHODS **********************/
 
 	// @codeCoverageIgnoreStart
```

Comment:
I'm missing a `@codeCoverageIgnoreEnd`. Might just be me tho...

Reply:
It's there all the way at the bottom: line 538.